webserver/templates/user/scraper.js

```diff
@@ -148,7 +148,10 @@ function getLastFMPage(page) {
 
 var version = "1.4";
 var page = 1;
-var numberOfPages = parseInt(document.getElementsByClassName("pages")[0].innerHTML.trim().split(" ")[3]);
+var numberOfPages = 1;
+if (document.getElementsByClassName("pages").length > 0) {
+    numberOfPages = parseInt(document.getElementsByClassName("pages")[0].innerHTML.trim().split(" ")[3]);
```

Comment:
It is the same reason why i used <code> queryselector </code>, as it just returned after fetching the first element in the container.